stdlib/config-api/src/main/ballerina/src/config/Module.md

```diff
@@ -3,7 +3,7 @@
 The `ballerina/config` module provides the Config API to read configurations from environment variables, TOML files, and command-line parameters, and build a consolidated set of configurations.
 
 The precedence order for configuration lookup is as follows: 
-1. CLI parameters (used with the -e flag)
+1. CLI configurations (used with the --)
```

Comment:
This should actually be CLI arguments right? CLI configurations sound like it's configs for the CLI

Reply:
+1 Replaced configurations with arguments.